### PR TITLE
#1693-E2E tests cleanup and local ad-hoc dev fixes

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Fixed `make forms`;
- Created `local-all` make command to give an idea of what needed to be executed to have a complete environment up and running;
- Fixed form.io address inside `docker-compose` that was causing the API to not reach the `formio` service;
- Fixed swagger API URL that was including `api` twice in the address;
- Form.io user and pass for deploying definitions is now using the **main .env** file.

### CODE REVIEW

File: sources/packages/backend/apps/api/src/main.ts [hunk 1]
```diff
@@ -69,7 +69,6 @@ async function bootstrap() {
         },
         "access-token",
       )
-      .addServer("/api")
```
Comment (dheepak-aot) [inline sources/packages/backend/apps/api/src/main.ts:72]: Adding screenshot to explain if anyone having doubts on this. 

image

**Review by dheepak-aot — APPROVED** [review]
Thanks for taking care of form.io as well. Nice work. 👍

File: sources/packages/backend/apps/api/src/main.ts (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/backend/apps/api/src/main.ts:72]: Thanks @dheepak-aot, please see below also the double `api` in the URL that was causing all calls to fail.
image

File: sources/Makefile [hunk 2]
```diff
@@ -62,12 +62,16 @@ export CRA_ENVIRONMENT_CODE := $(or $(CRA_ENVIRONMENT_CODE), )
 #Institution Integration
 export INSTITUTION_REQUEST_FOLDER := $(or $(INSTITUTION_REQUEST_FOLDER), Institution-Request)
 
+# Starts all applications: SIMS application (Web Portals, SIMS Api, Workers, Queue Consumers), 
+# and also Form.io server, and Camunda releated services.
+# Use make deploy-camunda-definitions to deploy all workflow definitions.
+# Use make deploy-form-definitions to deploy all form.io definitions.
+local-all: | print-status forms camunda local-build-run
```
Comment (andrepestana-aot) [inline sources/Makefile:69]: 👍

**Review by andrepestana-aot — APPROVED** [review]
Looks good!

**Pull request merged**